crawler/file_processing.py

```diff
@@ -833,7 +827,7 @@ def log_adding_default_lab_id(self, row, line_number):
         )
 
     def filtered_row(self, row, line_number) -> Dict[str, Any]:
-        """Filter unneeded columns and add lab id if not present and config flag set.
+        """Filter un-needed columns and add lab id if not present and config flag set.
```

Comment:
I think **_unneeded_** is the correct spelling

Reply:
🤣